Makefile

```diff
@@ -48,7 +48,7 @@ push-image:
 pull:
 	docker-compose -f docker-compose.yaml -f docker-compose.deps.yaml pull
 
-up:
+up: db-docker
```

Comment:
Why?

Reply:
because the internal server fails without having a DB first